assets-and-images.md

```diff
@@ -53,17 +57,30 @@ different locales or regions, reading directions, etc.
 
 ## Working with assets
 
-Your application can access its assets through the
-[AssetBundle](http://docs.flutter.io/flutter/services/AssetBundle-class.html)
-interface. The two main methods on an asset bundle allow you to load a string
+Your application can access its assets through an
+[AssetBundle](http://docs.flutter.io/flutter/services/AssetBundle-class.html).
```

Comment:
Suggest: "through an [`AssetBundle`](http://docs.flutter.io/flutter/services/AssetBundle-class.html) object."

Please use code fonts when referring to classes, methods, and constants.